README.md

```diff
@@ -58,6 +58,15 @@ For each test suite file...
 
 To see a working example check out the [Node.js test runner](https://github.com/adobe/target-nodejs-sdk/blob/main/packages/target-decisioning-engine/test/decisioning.spec.js)
 
+### Adding Additional Tests
+
+To add additional tests to the suite:
+1. Create a new activity in the `Target Testing` admin [here](https://experience.adobe.com/#/@targettesting/target/activities)
+1. Find the object that contains the `artifactFilename` you wish to update in `src/testArtifacts.json`. Update `activityIds` and `adminUrls` with the activity ID and admin URL of the new activity
+1. Follow the steps [below](#generating-updated-test-artifacts) to update the artifacts located at `src/schema.artifacts` with the new activity
+1. Update the corresponding test model located at `src/schema/models` with the new activity. Specifications for the test models can be found [here](#test-models)
```

Comment:
please mention on step 4 that this should be done by using the `updateTestSchema.sh` script mentioned in  #updating-to-the-latest-schema